src/tox_current_env/hooks3.py

```diff
@@ -0,0 +1,260 @@
+import os
```

Comment:
I would probably have used `git mv` to rename `src/tox_current_env/hooks.py` to `src/tox_current_env/hooks3.py` and thus keep the git history.

Reply:
Done in separated commit, thanks!